packages/builder-vite/svelte/csf-plugin.ts

```diff
@@ -23,22 +25,26 @@ export default function csfPlugin(svelteOptions?: Options) {
           .map(([id]) => `export const ${id} = __storiesMetaData.stories[${JSON.stringify(id)}];`)
           .join('\n');
 
-        const codeWithoutDefaultExport = code.replace('export default ', '// export default ');
+        s.replace('export default', '// export default');
 
         const namedExportsOrder = Object.entries<any>(stories)
           .filter(([, def]) => !def.template)
           .map(([id]) => id);
 
         const output = [
-          codeWithoutDefaultExport,
+          '',
```

Comment:
We could just remove this right?

Reply:
No, it's needed so that a newline is added.